### PR TITLE
Release 1.3.0

### PR DESCRIPTION
**Pull request by Zethson**
1.3.0 (2021-01-17)
------------------

**Added**

* cli-python version 2.0.0 with Poetry, Nox, pre-commit (with black, prettier, [...]), working module autogeneration and much more.

**Fixed**

**Dependencies**

**Deprecated**

### CODE REVIEW

**Comment by Zethson** [comment]
Could not load hsdis-amd64.so; library not loadable; PrintAssembly is disabled

https://askubuntu.com/questions/710392/java-8-oracle-1-8-0-66-problem-with-printassembly-could-not-load-hsdis-amd64

**Comment by Zethson** [comment]
What I do not remember: Did I protect the pytoml and poetry.lock files from sync?

**Comment by Zethson** [comment]
Check out whether we can get rid of labeler

https://github.com/theislab/spapros/pull/7

**Comment by Zethson** [comment]
Also, the tests badge didn't work.
However, it should now be possible to more easily hardcode the badge
eg

https://github.com/theislab/spapros/actions/workflows/run_tests.yml/badge.svg

**Comment by Zethson** [comment]
https://github.com/cjolowicz/cookiecutter-hypermodern-python/issues/791

**Comment by Zethson** [comment]
Should also add a max open PR limit for dependabot

**Comment by Zethson** [comment]
Incorporate https://github.com/cjolowicz/cookiecutter-hypermodern-python/pull/769/files

File: cookietemple/create/templates/cli/cli_python/{{ cookiecutter.project_slug_no_hyphen }}/mypy.ini [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[mypy]
+strict = True
```
Comment (Zethson) [inline cookietemple/create/templates/cli/cli_python/{{ cookiecutter.project_slug_no_hyphen }}/mypy.ini:2]: @Imipenem thinking about defaulting this to False.
It is really strict

File: cookietemple/create/templates/cli/cli_python/{{ cookiecutter.project_slug_no_hyphen }}/pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+name = "{{ cookiecutter.project_name }}"
+version = "{{ cookiecutter.version }}"  # <<COOKIETEMPLE_FORCE_BUMP>>
+description = "{{ cookiecutter.project_short_description }}"
+authors = ["{{ cookiecutter.full_name }} <{{ cookiecutter.email }}>"]
+license = "{{ cookiecutter.license }}"
+readme = "README.rst"
+homepage = "https://github.com/{{ cookiecutter.github_username }}/{{ cookiecutter.project_slug }}"
+repository = "https://github.com/{{ cookiecutter.github_username }}/{{ cookiecutter.project_slug }}"
+documentation = "https://{{ cookiecutter.project_slug }}.readthedocs.io"
+{% if cookiecutter.project_slug != cookiecutter.project_slug_no_hyphen -%}
```
Comment (Zethson) [inline cookietemple/create/templates/cli/cli_python/{{ cookiecutter.project_slug_no_hyphen }}/pyproject.toml:11]: I am not sure why I did not delete this if statement, but there might be a reason?

File: cookietemple/create/templates/common_files/{{cookiecutter.commonName}}/.github/workflows/master_branch_protection.yml [hunk 3]
```diff
@@ -0,0 +1,72 @@
+name: PR to master branch from patch/release branch only
+
+on:
+    pull_request:
+        branches:
+            - master
```
Comment (Zethson) [inline cookietemple/create/templates/common_files/{{cookiecutter.commonName}}/.github/workflows/master_branch_protection.yml:6]: missing main @Imipenem

File: docs/sync.rst [hunk 4]
```diff
@@ -56,6 +56,19 @@ Configuring sync
 
 .. _sync_config:
 
+Enable/Disable sync
+~~~~~~~~~~~~~~~~~~~
+
+Cookietemple aims to provide the user as much configuration as possible. So, the sync feature is optional and should also
+be switched on or off. If you want to enable sync (which is the default), the ``sync_enable`` accepts the following values: ``True, true, Yes, yes, Y, y``. To disable sync,
+simply change this value into ``False, false, No, no, N, n``. It can be configured in the::
```
Comment (Zethson) [inline docs/sync.rst:64]: one of

**Comment by Zethson** [comment]
I should also try to restrict the Python dependencies to >= and not ^

**Comment by Zethson** [comment]
Use python_versions for nox config

**Comment by Zethson** [comment]
Add try except for import of nox-poetry

**Comment by Zethson** [comment]
@Imipenem release-drafter must be removed from the cli-python template

File: cookietemple/bump_version/bump_version.py [hunk 5]
```diff
@@ -231,109 +225,19 @@ def check_bump_range(self, current_version: str, new_version: str) -> bool:
 
     def lint_before_bump(self) -> None:
         """
-        Lint the changelog prior to bumping. Linting consists of three major points.
-
-        1. Check, whether a CHANGELOG.rst file exist in top level directory of the project.
-        2. Lint CHANGELOG.rst to ensure that bump-version can safely add a new section
-        3. Check, whether all versions are consistent over the project
+        Lint the changelog prior to bumping. Check, whether all versions are consistent over the project
```
Comment (Imipenem) [inline cookietemple/bump_version/bump_version.py:228]: Remove `changelog` @Imipenem

**Pull request merged**